examples/meta/generator/generate.py

```diff
@@ -123,6 +123,7 @@ def translateExamples(inputDir, outputDir, targetsDir, ctagsFile,
         'csharp',
         'ruby',
         'lua',
+        'scala',
```

Comment:
I thought we don't generate scala code? But rather run compiled java examples using scala ?